new-lamassu-admin/src/components/expandable-table/ExpTable.js

```diff
@@ -28,7 +28,7 @@ const ExpRow = ({ id, columns, details, sizes, expanded, className, expandRow, .
 
```

Comment:
This component has some serious performance issues, I'll try to lay it all out here. Some comments are not on the right place because I can only review the current PR